src/widgets/SearchBar.js

```diff
@@ -37,30 +40,49 @@ export const SearchBar = ({
   debounceTimeout,
   ...textInputProps
 }) => {
+  const [textValue, setTextValue] = useState('');
```

Comment:
I thought original problem was that the old search bar had state? useEffect and other tweaks fix the prob?

Reply:
I  guess I named it incorrectly. It was more that the component was fully uncontrolled